src/main/java/de/hhu/stups/plues/ui/components/MajorMinorCourseSelection.java

```diff
@@ -191,6 +201,33 @@ private void filterCbMinorCourses() {
     cbMinor.getSelectionModel().select(0);
   }
 
+  private void fireListenerEvents() {
+    for (ChangeListener listener : listeners) {
+      listener.changed(this, null, null);
+    }
+  }
+
+  @Override
+  public void addListener(ChangeListener listener) {
+    listeners.add(listener);
+  }
+
+  @Override
+  public void removeListener(ChangeListener listener) {
+    listeners.remove(listener);
+  }
+
+  @Override
```

Comment:
Maybe just implement observable (for now) to avoid returning null when reading the value.